examples/proc_Ep.py

```diff
@@ -32,7 +32,6 @@
     )
     orig_axis = s["indirect"]  # let's save this so we
     #                           can pass it to the log
-    s = prscr.convert_to_power(thisfile, exptype, s, fl=fl)
```

Comment:
I just moved this to the end so that we could see the older plot (which I thought was kind of nice) as well as the final plot.